### ISSUE TITLE
Add ToneDen.io Support

### ISSUE THREAD

**Issue by Miserlou** [Miserlou/SoundScrape#75]
Man I hate that website.

**Comment by pydo** [comment]
Do they even have an api for free downloads? I quickly checked their site. To download tracks I had to authenticate myself with some 3rd party social media services. 
Wouldn't this require phantomjs scraping with access to accounts like spotify?
Seems like a lot of work I'm wondering if it's worth it.

**Comment by Miserlou** [comment]
Yeah, it will likely require acquiring some fake credentials or using a service like Mailinator.
